clients/drcachesim/common/options.cpp

```diff
@@ -1035,5 +1043,12 @@ droption_t<bool> op_abort_on_invariant_error(
     "total invariant error count is printed at the end; a non-zero error count does not "
     "affect the exit code of the analyzer.");
 
+droption_t<bool> op_pt2ir_allow_recoverable_errors(
```

Comment:
Same comment about "recoverable" as above.  How about "allow_discontinuities": that seems much more specific and understandable and lets the user know what the consequences are.

DItto below.

Reply:
I think it'll be better to rename this to op_pt2ir_best_effort_conversion, which would cover not just these 1-instr discontinuities but also the total failure in syscall conversion.

Reply:
Or alternatively op_pt2ir_ignore_errors if you think that's clearer.